quickfixj-core/src/main/java/quickfix/SessionState.java

```diff
@@ -260,6 +259,10 @@ public MessageStore getMessageStore() {
         return messageStore;
     }
 
+    public MessageQueue getMessageQueue() {
+        return messageQueue;
```

Comment:
Is there a specific reason why you have a getter for the `messageQueue` and then do `state.getMessageQueue().xyz` all the time?

Reply:
I wanted to make usage the same as you do with message store:

Sessions does `state.getMessageStore().xyz` calls

Would you prefer to keep those methods?

Reply:
I see, let me think about it...